corehq/apps/userreports/ui/widgets.py

```diff
@@ -5,4 +5,8 @@
 class JsonWidget(forms.Textarea):
 
     def render(self, name, value, attrs=None):
+        if isinstance(value, basestring):
+            # It's probably invalid JSON
```

Comment:
I initially had this condition be basically `if isinstance(value, basestring) and not is_json(value):`.  Then I noticed an attempt is made to coerce this to json here: https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/userreports/ui/fields.py#L30-L37, so I dropped that part.
```python
In [2]: a = "{'foo': 'bar'}"

In [3]: json.dumps(a)
Out[3]: '"{\'foo\': \'bar\'}"'

In [4]: a
Out[4]: "{'foo': 'bar'}"

In [5]: b = "this is just a string"

In [6]: json.dumps(b)
Out[6]: '"this is just a string"'

In [7]: b
Out[7]: 'this is just a string'

In [8]: json.loads('"foo"')
Out[8]: u'foo'

In [9]: json.dumps(json.loads('"foo"'))
Out[9]: '"foo"'
```
Strings are `basestring`s, obviously, and they can be dumped to JSON.  This could run in to some weirdness if you allow unwrapped string values.  Maybe I should add back in the `not is_json` part?